sdk/Lusid.Sdk.Tests/tutorials/Instruments/ComplexInstruments.cs

```diff
@@ -517,6 +517,53 @@ public void DemonstrateCreationOfEquityOption()
             Assert.That(roundTripEquityOption.UnderlyingIdentifier, Is.EqualTo(roundTripEquityOption.UnderlyingIdentifier));
         }
 
+        [Test]
+        public void DemonstrateCreationOfTermDeposit()
+        {
+            // CREATE a new TermDeposit
+            var termDeposit = new TermDeposit(
+                startDate: new DateTimeOffset(2020, 2, 5, 0, 0, 0, TimeSpan.Zero),
+                maturityDate: new DateTimeOffset(2020, 8, 5, 0, 0, 0, TimeSpan.Zero),
+                contractSize: 1_000_000m,
+                flowConvention: new FlowConventions(
+                    scope: null,
+                    code: null,
+                    currency: "GBP",
+                    paymentFrequency: "6M",
+                    rollConvention: FlowConventions.RollConventionEnum.MF,
+                    dayCountConvention: FlowConventions.DayCountConventionEnum.Act365,
+                    holidayCalendars: new List<string>(),
+                    settleDays: 1,
+                    resetDays: 0
+                ),
+                rate: 0.03m,
+                instrumentType: LusidInstrument.InstrumentTypeEnum.TermDeposit
+            );
+
+            // ASSERT that it was created
+            Assert.That(termDeposit, Is.Not.Null);
```

Comment:
this seems a bit superfluous, is there a case where it can be `null` without `new TermDeposit(...)` throwing an exception?

Reply:
No, you're right. Removed.